### DISCUSSION TITLE
Is this built on top of Tk or as a branch from it.

### DISCUSSION THREAD

**Discussion by jeffdn2** [TomSchimansky/CustomTkinter#657] [General]
When new versions of Tkinter are released will they be supported with CustomTkinter automatically?

**Comment by TomSchimansky** [comment]
It's built on top of tkinter. The widgets all inherit from tkinter.Frame and create a canvas in this frame to render the widgets. As long as tkinter updates are backwards compatible CustomTkinter will support them I would say.

**Comment by jeffdn2** [comment]
-Sent by Jeff Duncan mobile (615) 934-1291On Nov 21, 2022, at 3:46 PM, Tom Schimansky ***@***.***> wrote:﻿
It's built on top of tkinter. The widgets all inherit from tkinter.Frame and create a canvas in this frame to render the widgets. As long as tkinter updates are backwards compatible CustomTkinter will support them I would say.
